Sources/SwiftLayout/Core/Activator/Activator.swift

```diff
@@ -15,11 +15,7 @@ enum Activator {
 
     @discardableResult
     static func update<L: Layout>(layout: L, fromActivation activation: Activation, forceLayout: Bool) -> Activation {
-        var prevInfos: [ViewInformation: Set<WeakConstraint>] = [:]
-        for viewInfo in activation.viewInfos {
-            guard let view = viewInfo.view else { continue }
-            prevInfos[viewInfo] = Set(view.constraints.weakens)
-        }
+        let prevInfoHashValues = Set(activation.viewInfos.map(\.hashValue))
```

Comment:
oh! I didn't think of that part. I'll think about a good way.

However, it seems that the same problem may occur with the method implemented as a dictionary before.

Reply:
> oh! I didn't think of that part. I'll think about a good way.
> 
> However, it seems that the same problem may occur with the method implemented as a dictionary before.

so right.

Reply:
@oozoofrog 
I thought again,
Since a new set is created using the hashValue value, the above-mentioned problem does not seem to occur even in a situation where all hashValues ​​have the same value.

Reply:
@gmlwhdtjd OMG, i didn't see that.